### PR TITLE
Remove sql like function in base_hook

### PR DESCRIPTION
**Pull request by zhongjiajie**
---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by zhongjiajie** [comment]
oh, have a bad rebaes

File: UPDATING.md [hunk 1]
```diff
@@ -216,15 +221,6 @@ The following methods were moved:
 | airflow.providers.google.cloud.hooks.bigquery.BigQueryBaseCursor.run_with_configuration        | airflow.providers.google.cloud.hooks.bigquery.BigQueryHook.run_with_configuration        |
 | airflow.providers.google.cloud.hooks.bigquery.BigQueryBaseCursor.update_dataset                | airflow.providers.google.cloud.hooks.bigquery.BigQueryHook.update_dataset                |
 
-### Make behavior of `none_failed` trigger rule consistent with documentation
-The behavior of the `none_failed` trigger rule is documented as "all parents have not failed (`failed` or
```
Comment (potiuk) [inline UPDATING.md]: I thin that was removed by accident?

**Review by turbaszek — APPROVED** [review]

**Comment by zhongjiajie** [comment]
Seem unrelated failed 
```sh
self = <tests.cli.commands.test_task_command.TestCliTasks testMethod=test_local_run>
    def test_local_run(self):
        args = self.parser.parse_args([
            'tasks',
            'run',
            'example_python_operator',
            'print_the_context',
            '2018-04-27T08:39:51.298439+00:00',
            '--interactive',
            '--subdir',
            '/root/dags/example_python_operator.py'
        ])
    
        dag = get_dag(args.subdir, args.dag_id)
        reset(dag.dag_id)
    
>       task_command.task_run(args)
tests/cli/commands/test_task_command.py:237: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
airflow/cli/commands/task_command.py:183: in task_run
    _run_task_by_selected_method(args, dag, ti)
airflow/cli/commands/task_command.py:63: in _run_task_by_selected_method
    _run_task_by_executor(args, dag, ti)
airflow/cli/commands/task_command.py:99: in _run_task_by_executor
    executor.end()
airflow/executors/local_executor.py:326: in end
    self.impl.end()
airflow/executors/local_executor.py:285: in end
    self.queue.join()
<string>:2: in join
    ???
/usr/local/lib/python3.6/multiprocessing/managers.py:757: in _callmethod
    kind, result = conn.recv()
/usr/local/lib/python3.6/multiprocessing/connection.py:250: in recv
    buf = self._recv_bytes()
/usr/local/lib/python3.6/multiprocessing/connection.py:407: in _recv_bytes
    buf = self._recv(4)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <multiprocessing.connection.Connection object at 0x7f08c54e6c18>
size = 4, read = <built-in function read>
    def _recv(self, size, read=_read):
        buf = io.BytesIO()
        handle = self._handle
        remaining = size
        while remaining > 0:
            chunk = read(handle, remaining)
            n = len(chunk)
            if n == 0:
                if remaining == size:
>                   raise EOFError
E                   EOFError
/usr/local/lib/python3.6/multiprocessing/connection.py:383: EOFError
```

**Comment by zhongjiajie** [comment]
Still failing PG test 😿 , will try in local with breeze.

**Comment by zhongjiajie** [comment]
Passing in breeze, will try to force-push to restart the failing test.

**Review by feluelle — APPROVED** [review]

**Comment by zhongjiajie** [comment]
Still failing......

**Comment by zhongjiajie** [comment]
@turbaszek @feluelle CI green, please take a look

**Review by potiuk — APPROVED** [review]
LGTM

**Comment by potiuk** [comment]
Thanks @zhongjiajie !

File: UPDATING.md (hunk 1 above)
Comment (zhongjiajie) [reply UPDATING.md]: Just a bad rebase and already fix it.

**Comment by zhongjiajie** [comment]
Thanks for merging

**Pull request merged**